package.json

```diff
@@ -2,5 +2,9 @@
   "name": "memes_chat",
   "devDependencies": {
     "prettier": "^2.8.2"
+  },
+  "dependencies": {
```

Comment:
What's going on here, why u save deps in project root package??

Reply:
Idk why this happend :/